web3.js/src/connection.ts

```diff
@@ -63,6 +63,13 @@ export const BLOCKHASH_CACHE_TIMEOUT_MS = 30 * 1000;
 
 type RpcRequest = (methodName: string, args: Array<any>) => any;
 
+type RpcBatchRequest = (requests: RpcParams[]) => any;
+
+export type RpcParams = {
```

Comment:
Can you please add an `@internal` comment tag to this type to avoid exposing it on the doc site?
![Screen Shot 2021-03-22 at 2 16 34 PM](https://user-images.githubusercontent.com/1076145/111947921-456ac300-8b19-11eb-9350-a1f23144d627.png)